tests/CoreTest.php

```diff
@@ -274,7 +274,7 @@ public function typeProvider()
      */
     public function testDescribesType($input, $output)
     {
-        $this->assertEquals($output, Core::describeType($input));
+        $this->assertContains($output, Core::describeType($input));
```

Comment:
Why is this change necessary?

Reply:
Because of the way PHP outputs the content with var_dump from 7.x on with information about the file and line number where the var_dump has been called. E.G.:

```
5) GuzzleHttp\Tests\Ring\CoreTest::testDescribesType with data set #4 (1.0, 'float(1)')
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
+RingPHP/src/Core.php:301:
 float(1)

RingPHP/tests/CoreTest.php:277
```

And Core::describeType uses var_dump to output information about the input

Reply:
I can't recall any changes to var_dump, also can't find anything in the documentation.

Also can't reproduce it here: https://3v4l.org/jXMF6

Is there a chance that it's caused by xdebug var_dump overloading? That's my guess at least.

Reply:
Oh man. You are so right. The line output started for me with PHP7. But yeah it was the xdebug var_dump overload. Maybe they introduced this feature with the php7 release. I will revert this change.